package.json

```diff
@@ -38,6 +38,7 @@
         ]
     },
     "dependencies": {
-        "dropzone": "^5.5.1"
+        "dropzone": "^5.5.1",
```

Comment:
Why do you need `dropzone` package? I didn't see any place where you would use it.